tests/test_examples.py

```diff
@@ -1534,7 +1542,11 @@ def test_frozen_environment(tmp_path, request):
             assert frozen_file.exists()
             actual_frozen_paths.add(frozen_file)
 
-        assert expected_frozen_paths == actual_frozen_paths, (
-            f"Expected: {sorted(str(p) for p in expected_frozen_paths)}\n"
+        assert set(expected_frozen_paths.values()) == actual_frozen_paths, (
```

Comment:
You don't need to handcraft your message here. `assert set(...) == set(...)` already has a very nice diff that will be easier to read in the log than comparing "Expected" and "Found" (where comparisons will have to be done "by hand").